huaweicloud/services/eip/resource_huaweicloud_vpc_bandwidth.go

```diff
@@ -268,6 +333,14 @@ func waitForBandwidth(client *golangsdk.ServiceClient, id string) resource.State
 	}
 }
 
+func normalizeChargingMode(billing string) string {
+	mode := "postPaid"
+	if billing != "" {
+		mode = "prePaid"
+	}
+	return mode
+}
```

Comment:
```suggestion
func normalizeChargingMode(billing string) string {
	if billing != "" {
		return "prePaid"
	}
	return "postPaid"
}
```
In this way, one less memory space can be created and one less memory operation can be reduced.